dora/core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -7944,17 +7362,6 @@ public String toString() {
       booleanBuilder("alluxio.test.deprecated.key")
           .build();
 
-  /**
-   * @param fullyQualifiedClassname a fully qualified classname
-   * @return html linking the text of the classname to the alluxio javadoc for the class
-   */
-  private static String javadocLink(String fullyQualifiedClassname) {
```

Comment:
documentation related things can be preserved?

Reply:
Previously it was utilized to generate a description for a certain property key. However, the property key has been removed, and there is currently no place using this method.

Reply:
Do you think they might be used again some day? If not then we can remove.

Reply:
I think nobody know this this method, even if we preserve it, it would may not be used.